server/server.go

```diff
@@ -142,7 +142,7 @@ func (c *server) initialize(ctx context.Context) error {
 	conf := config.GetGlobalServerConfig()
 	subscriptionClient := logpuller.NewSubscriptionClient(
 		&logpuller.SubscriptionClientConfig{
-			RegionRequestWorkerPerStore: 8,
```

Comment:
Please do not change it until a full test is conducted to verify that it will not impact performance.

Reply:
This PR won't be merged, after all other signification problems get fixed. When merging this PR, we should also care about the performance of the log puller won't regressed.